js/lib/ui.js

```diff
@@ -1,472 +1,484 @@
 elgg.provide('elgg.ui');
 
```

Comment:
This is the most unusual conversion. For BC I wanted to keep defining the functions synchronously, but load the translators via require(), before calling the init. I see no other way than to drag these functions inside a shared scope.